services/QuillLMS/engines/evidence/lib/evidence/evidence/feedback_assembler.rb

```diff
@@ -33,6 +33,9 @@ def self.default_payload
         'rule_uid': ''
       }  
     end
-  end
-    
+
+    def self.error_name 
+      'abstract_error'
```

Comment:
Maybe just `raise NoMethodError` or something here in the abstract class?  That'll actually cause errors if someone fails to do a sub-class implementation which I think is the behavior we want?